component/service/pom.xml

```diff
@@ -295,6 +295,10 @@
         <artifactId>jersey-client</artifactId>
         <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.exoplatform.social</groupId>
+          <artifactId>social-component-webui</artifactId>
```

Comment:
The Rest Services shouldn't depend on Webui artifact

Reply:
@boubaker  This was fixed, could you re-validate please :)